### PR TITLE
ES6 and prerequisite to shortcut editor

### PR DESCRIPTION
**Pull request by Carreau**
For whatever reason casper seem to be unhappy when I actually introduce es6 things in modules, 
so this is a subset of #1347 where tests are (or at least should be) passing on travis.

### CODE REVIEW

File: notebook/static/notebook/js/keyboardmanager.js [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 define([
     'base/js/utils',
-    'base/js/keyboard',
+    'base/js/keyboard'
```
Comment (minrk) [inline notebook/static/notebook/js/keyboardmanager.js]: Stripping trailing commas makes me a little sad. Just curious: what brought this on?

Comment (Carreau) [inline notebook/static/notebook/js/keyboardmanager.js]: Sorry, tried enabling `eslint` and my editor complains. I can deactivate and revert that.

**Comment by Carreau** [comment]
Fixed.

**Comment by Carreau** [comment]
Yeah ! Thanks !

**Pull request merged**